cmd/manager/main.go

```diff
@@ -168,6 +168,11 @@ func init() {
 		"",
 		fmt.Sprintf("K8s secret mounted into the path designated by %s to be used for webhook certificates", operator.WebhookCertDirFlag),
 	)
+	Cmd.Flags().String(
+		operator.WebhookConfigurationNameFlag,
+		DefaultWebhookConfigurationName,
+		fmt.Sprintf("Name of K8s ValidatingWebhookConfiguration to be used for webhook params (defaults to %s)", DefaultWebhookConfigurationName),
```

Comment:
I would maybe rephrase to `Name of the Kubernetes ValidatingWebhookConfiguration resource` to be more explicit as a flag. People don't need to know about the "webhook params" which is our own internal struct in the code.

Reply:
Also maybe a note along the lines of "only used when `enable-webhook` is `true`"

Reply:
@sebgl , @anyasabo thanks for your PR review.
I rephrased as suggested: `Name of the Kubernetes ValidatingWebhookConfiguration resource (defaults to %s). Only used when enable-webhook is true.`